stock_picking_assign_serial_final/wizards/stock_assign_serial_views.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_assign_serial_numbers" model="ir.ui.view">
+        <field name="model">stock.assign.serial</field>
+        <field name="inherit_id" ref="stock.view_assign_serial_numbers" />
```

Comment:
Add it to `stock.view_stock_move_operations` view as well for operation type with no detailed operations checked:

![image](https://user-images.githubusercontent.com/5040182/165914080-161abea2-287a-4e31-903c-39c41e947b51.png)